rocAL/rocAL/include/loaders/image/node_image_loader.h

```diff
@@ -41,7 +41,7 @@ class ImageLoaderNode : public Node
     /// The loader will repeat images if necessary to be able to have images in multiples of the load_batch_count,
     /// for example if there are 10 images in the dataset and load_batch_count is 3, the loader repeats 2 images as if there are 12 images available.
     void init(unsigned internal_shard_count, const std::string &source_path, const std::string &json_path, const std::map<std::string, std::string> feature_key_map, StorageType storage_type, DecoderType decoder_type, bool shuffle, bool loop,
-              size_t load_batch_count, RocalMemType mem_type, std::shared_ptr<MetaDataReader> meta_data_reader, bool decoder_keep_orig = false, const char *prefix = "", unsigned sequence_length = 0, unsigned step = 0, unsigned stride = 0);
+              size_t load_batch_count, RocalMemType mem_type, std::shared_ptr<MetaDataReader> meta_data_reader, bool decoder_keep_orig = false, FileMode file_mode = FileMode::FILENAME, const char *prefix = "", unsigned sequence_length = 0, unsigned step = 0, unsigned stride = 0);
```

Comment:
Choose a consistent function parameter name for the file mode - file_mode and external_file_mode are used in two different files